package-lock.json

```diff
@@ -79,11 +79,11 @@
       }
```

Comment:
I don't think that the package lock should have changed without any package json file changes